tests/estimator/ops/test_estimator_symbolics.py

```diff
@@ -184,6 +239,111 @@ def controlled_resource_decomp(
+                qre.Resources(
+                    zeroed=5,
+                    any_state=0,
+                    algo_wires=12,
+                    gate_types=defaultdict(
+                        int,
+                        {
+                            resource_rep(qre.CNOT): 32,
+                            resource_rep(qre.Toffoli, {"elbow": "left"}): 8,
+                            resource_rep(qre.Toffoli): 2,
+                            resource_rep(qre.Hadamard): 24,
+                        },
+                    ),
+                ),
+            ),
+            (
+                qre.CRZ(precision=1e-3),
+                qre.Resources(
+                    zeroed=0,
+                    any_state=0,
+                    algo_wires=3,
+                    gate_types=defaultdict(
+                        int,
+                        {
+                            resource_rep(qre.CNOT): 4,
+                            resource_rep(qre.T): 84,
+                            resource_rep(qre.Toffoli): 2,
+                        },
+                    ),
+                ),
+                qre.Resources(
+                    zeroed=1,
+                    any_state=0,
+                    algo_wires=4,
+                    gate_types=defaultdict(
+                        int,
+                        {
+                            resource_rep(qre.CNOT): 2,
+                            resource_rep(qre.T): 84,
+                            resource_rep(qre.Toffoli): 6,
+                            resource_rep(qre.Toffoli, {"elbow": "left"}): 2,
+                            resource_rep(qre.Hadamard): 6,
+                        },
+                    ),
+                ),
+            ),
+            (
+                qre.CRZ(),
+                qre.Resources(
+                    zeroed=0,
+                    any_state=0,
+                    algo_wires=3,
+                    gate_types=defaultdict(
+                        int,
+                        {
+                            resource_rep(qre.CNOT): 4,
+                            resource_rep(qre.T): 176,
+                            resource_rep(qre.Toffoli): 2,
+                        },
+                    ),
+                ),
+                qre.Resources(
+                    zeroed=1,
+                    any_state=0,
+                    algo_wires=4,
+                    gate_types=defaultdict(
+                        int,
+                        {
+                            resource_rep(qre.CNOT): 2,
+                            resource_rep(qre.T): 176,
+                            resource_rep(qre.Toffoli): 6,
+                            resource_rep(qre.Toffoli, {"elbow": "left"}): 2,
+                            resource_rep(qre.Hadamard): 6,
+                        },
+                    ),
+                ),
+            ),
+        ),
+    )
+    def test_estimate_resource_decomp(self, base_op, ctrl_res, ctrl_ctrl_res):
```

Comment:
Same here for rename. Right now these tests will just autopass.

Reply:
If they're under the same scope, the tests with the same name will just get redefined and only the last one will be actually executed.